dist/selleckt-legacy.js

```diff
@@ -861,21 +861,38 @@ _.extend(SingleSelleckt.prototype, {
         matchTerm = term.toLowerCase();
 
         return _.reduce(items, function(memo, item){
-            var matchIndex;
+            var matchIndex, matchStart, matchEnd;
 
             if (!item.value) {
                 return memo;
             }
 
             matchIndex = item.label.toLowerCase().indexOf(matchTerm);
+            matchStart = matchIndex;
+            matchEnd = matchIndex + (term.length - 1);
+
+            if (matchIndex === -1 && item.aliases && item.aliases.length > 0) {
+                // try the aliases
+                matchIndex = item.aliases.reduce(function(index, alias) {
+                    if (index === -1) {
+                        return String(alias).toLowerCase().indexOf(matchTerm);
```

Comment:
Is this going to cause weird issues when an `alias` is somehow `null` and you're searching for `nu`?